cdc/model/sink.go

```diff
@@ -253,7 +254,8 @@ type RowChangedEvent struct {
 
 	RowID int64 `json:"row-id" msg:"-"` // Deprecated. It is empty when the RowID comes from clustered index table.
 
-	Table *TableName `json:"table" msg:"table"`
+	Table    *TableName         `json:"table" msg:"table"`
+	ColInfos []rowcodec.ColInfo `json:"column-infos" msg:"-"`
```

Comment:
Please use `column_infos`.

Reply:
Sorry I don't really get it. Do you mean change the `json:"column-infos"` to `json:"column_infos"`? But all other fields use hyphens instead of underscores.

Reply:
`msg:"-"` seems to prevent the information from being persisted for Redo. Could you verify the compatibility of this change with Redo?

Reply:
Redo only applies to mysql sink currently so does none of avro business. So prevent `ColInfos` from being persisted for Redo seems right. 

But I think the `ColInfos` is important and maybe in future Redo could support avro? Persist `ColInfos` seems bring no trouble. Upgrade is also safe since a `nil` `ColInfos` field is safe for other codecs. But as @liuzix tells me, it will make the redo log large. So I'd like to leave it unpersist in this pull request. 

Could you take a look @Ben1009 ?

Reply:
lgtm for redo part